src/routes/blog/index.svelte

```diff
@@ -15,22 +15,31 @@
 </script>
 
 <svelte:head>
-  <title>acmCSUF / README</title>
+  <title>Blog / ACM at CSUF</title>
```

Comment:
Why "Blog" instead of "README"

Reply:
I'll go ahead and change that in the next update